pupynere.py

```diff
@@ -1135,14 +1137,14 @@ def byteorderer(target):
     except StopIteration:
         pass
 
-def check_byteorder(input):
+def check_byteorder(input_):
     ''' an iteration-based approach to byteorderer above
         input should be an iterable of numpy arrays with which to
         fill the netcdf file
     '''
     try:
         while True:
-            data = input.next()
+            data = input_.next()
```

Comment:
In Python3, I think that one is supposed to use the [`next()` builtin function](https://docs.python.org/3/library/functions.html#next). Should we start doing that now?